Gemfile

```diff
@@ -19,6 +19,7 @@ gem 'flipper', '~> 0.28.3'
 gem 'flipper-active_record', '~> 0.28.0'
 gem 'flipper-ui', '~> 0.28.0'
 gem 'jbuilder', '~> 2.10'
+gem 'lograge', '~> 0.13.0'
```

Comment:
Hadn't you recommended us to specify the version with 2 digits and allow minor updates?

Reply:
good point. From https://semver.org/ it says:
>MINOR version when you add functionality in a backward compatible manner

so it should be safe to do that yes. I'll change it